src/auto-evo/simulation/SimulationCache.cs

```diff
@@ -320,7 +321,11 @@ public float GetPredationScore(Species species, Species preySpecies, IBiomeCondi
             scoreMultiplier *= Constants.AUTO_EVO_CHUNK_LEAK_MULTIPLIER;
         }
 
-        cached = (scoreMultiplier * behaviourScore * (pilusScore + engulfScore + oxytoxyScore + predatorSlimeJetScore) -
+        var enzymesScore = predatorEnzymes
+            .Sum(enzyme => Constants.AutoEvoLysosomeEnzymesScores.TryGetValue(enzyme, out var value) ? value : 0.0f);
+
+        cached = (scoreMultiplier * behaviourScore *
+                (pilusScore + engulfScore + oxytoxyScore + predatorSlimeJetScore + enzymesScore) -
                 (preySlimeJetScore + preyMucocystsScore)) /
```

Comment:
realistically this should probably be a multiplier on engulf score instead of just a flat score added on